client/src/app/services/rsvp.service.ts

```diff
@@ -10,4 +10,8 @@ export class RsvpService {
   sendRSVP(data: RecursivePartial<submissionData>) {
 	return this.api.post(`rsvp`, data);
   }
+
+  getRSVPFromCode(code: string) {
```

Comment:
Is there a reason for "code" being generic - it's listed _as_ their pokemon iirc. Super nitpicky, but should we rename that?